### ISSUE TITLE
ping

### ISSUE THREAD

**Issue by kykyi** [RustSydney/talks#43]
Is there interest in getting this user group started again?

**Comment by thomaseizinger** [comment]
I'd love a Rust meetup again! Also happy to help out organising if it needs although I cannot help with a venue if we want to do it in person.

**Comment by pinkforest** [comment]
keen

**Comment by pinkforest** [comment]
just need someone who is active and has the bits for the meetup group i guess?
I could organise the next one if there is nobody else

**Comment by pinkforest** [comment]
@caspark @srijs @bitshifter @joshleeb ping :)

**Comment by caspark** [comment]
Hello! I haven't lived in Sydney for the last 5 years, so I handed off admin privileges for the [Rust Sydney meetup group](https://www.meetup.com/Rust-Sydney/) and this github org a while ago. Might want to try pinging [the current leaders on meetup](https://www.meetup.com/rust-sydney/members/?op=leaders) to raise them there?

(I've pinged the one Atlassian in the meetup leadership group in Atlassian's Slack instance too.)

**Comment by bitshifter** [comment]
I am also no longer in Sydney but still have access to the Meetup group. As Caspar suggested you are better off asking there. I don't know that our old hosts are doing in person meetups yet, I know my Sydney based office isn't. If you did want to organise hostng yourself, I could assist with the MeetUp app side, best to have that conversation via MeetUp, not all of the organisers are active in GitHub.

**Comment by thomaseizinger** [comment]
@pinkforest Are you a member of the meet-up group? I can't find you with your nick unfortunately. Give me a ping there. Let's set up a group chat there with the current hosts to discuss how we can set up a new event.

**Comment by pinkforest** [comment]
@thomaseizinger I am now :unicorn:  MissMissM thats my medium @ handle - I used to run some events before and happy to organise

**Comment by pinkforest** [comment]
@bitshifter is there a way to give bits so I can create an event after I've secured a space ?

**Comment by pinkforest** [comment]
who is the current admin of the RustSydney org that could invite to the org so we can manage the talks repo?

**Comment by pinkforest** [comment]
who is the current admin of the RustSydney Github org that could invite to the org so we can manage the talks repo?

or could create another repo by month or something and then archive the oldies and then grant repo access

I would like to have a similar process as this week in rust for call for papers perhaps

**Comment by thomaseizinger** [comment]
> or could create another repo by month or something and then archive the oldies and then grant repo access

That sounds like quite the overhead and makes things like subscribing hard.

> I would like to have a similar process as this week in rust for call for papers perhaps

To start with, we could stick to the [original process](https://github.com/RustSydney/talks#proposing-a-talk) and perhaps message TWIR to get a shout-out on Twitter and in the newsletter or something so people are aware that this meet-up group is coming back to life?

**Comment by pinkforest** [comment]
Yes it is overhead but I was wondering if it helps getting bits as it's just one repo.

Another option is to create a new organisation as it seems to be the admin is not present.

I can do the TWIR part

I do not want long talks so I would be keen to limit talks to five - or max ten - minutes to make more varied talks possible and make sure people keep the talks compact and are focused on important bits - like lightning talks

**Comment by bitshifter** [comment]
I think I'm an admin. The GitHub is not super important though, everything was organised through Meetup in the past.

If you have a time and location for a Meetup sorted I can post it for you and talk to the others about adding organisers to Meetup group.

For TWiR I think they scrape meetups from the [Rust community calendar](https://www.google.com/calendar/render?cid=YXBkOXZtYmMyMmVnZW5tdHU1bDZjNWpiZmNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&invEmailKey=banderson@mozilla.com:7cf631a78192cbe527787baa308ae610b9548cb6&pli=1).

**Comment by pinkforest** [comment]
We are organising via Github on 19 July 2022 6PM AEST that has virtual component to it:
https://github.com/RustAU/Virtual

I will be securing a venue so this is a hybrid kind and then I will gate Sydney RSVP here:
https://github.com/RustAU/Sydney

I am a bit against paying for MeetUp.com when we can just send pull requests for RSVP'ing esp when it's just bunch of coders attending who often have Github :o

If you could post about the 19 July 2022 virtual lightning talk event with links to the Github repo that would be awesome

I've posted it to next TWIR

**Comment by bitshifter** [comment]
Amazon are paying for Meetup. The Rust Sydney Meetup group has hundreds of members, IMO forking the meetup is counter productive.

**Comment by pinkforest** [comment]
I agree and I would prefer that but problem is I don't have necessary access to manage participation and give updates.

My worry is that if I have a venue for 50 people signup via meetup;.com and 5 people show up then we end up wasting resources from venue PoV.

I would need a bit to manage participation to ensure we will not have no-shows and I thought GitHub would have allowed to do that via PRs.

How was the participation two years ago via meetup? Did you have to manage it actively? Is it a problem for this group?

If the participation is not a problem then happy to organise the participation via MeetUp and leverage the existing network.

We also started a discord for Rust AU / NZ folk in case anyone is interested - https://discord.gg/pW35BNSBeV

**Comment by bitshifter** [comment]
The discord link didn't work for me, it says invite expired, it might be easier to discuss this over there.

The last few meetups before the pandemic had 50 to 60 people attend. It is hard to say what attendance would be like now. I would guess lower.

I'm not sure what you mean by actively manage participation?

Meetup maintains an RSVP list, people can RSVP via Meetup and the organiser can get a list of attendees beforehand. Typically not all people who RSVP show up.

I would create an event via Meetup.com, add it to the Rust community calendar so that it would make it into TWiR and send a reminder out via Meetup a few days before the event. That's about it. The other organisers would handle the venue, food and drink. This was usually provided by Atlassian or Canva. I don't think either are doing in-person meetups again yet, neither is my workplace. We mostly had people organised to speak beforehand, they would volunteer via GitHub or by contacting one of the organisers, usually via Meetup.

**Comment by pinkforest** [comment]
Try this - not sure what happened to the link:
https://discord.gg/pW35BNSBeV

**Comment by anguslees** [comment]
I'm still trying to catch up.  If I'm reading correctly, we've discarded the existing organisation/proposal process, the existing github repo, and the existing meetup group - is that correct?   Feeling a bit steam-rolled, but I'm keen to help out where there's still room.

**Comment by sjwaight** [comment]
Hey everyone. If you're looking for a venue the Sydney Microsoft Reactor (in the Sydney Startup Hub at Wynyard) is available for communities to use. July might be a stretch, but happy to see how we can help out. Feel free to drop me a line at firstname . lastname @ microsoft[dot]com and we can work something out.
